content/en/announcements/_index.md

```diff
@@ -2,4 +2,6 @@
 title: Announcements
 cascade:
   type: docs
+  params:
+    hide_feedback: true
```

Comment:
@svrnm @tiffany76 - note that I've turned off the page feedback form for announcements.

Reply:
Not that folks should be looking at those pages, but then again, it's another reason to drop the form.